src/pages/resultsView/cancerSummary/CancerSummaryContent.tsx

```diff
@@ -805,6 +805,11 @@ export class CancerSummaryContent extends React.Component<
                                     this.hideGenomicAlterations
                                 }
                                 xLabels={this.chartData.labels}
+                                xAxisString={
+                                    _.keyBy(GroupByOptions, 'value')[
```

Comment:
we try to avoid using these lodash shortcuts (property name as string) b/c it's not typesafe.  instead ```(opt)=>opt.value)```
However, I would just use GroupOptions.find(...) which is more succinct.

Reply:
Changed from LoDash to array find